app/admin.py

```diff
@@ -19,7 +19,7 @@ class GirlAdmin(admin.ModelAdmin):
 
 
 admin.site.register(Girl, GirlAdmin)
-
+'rest_framework','rest_framework',
```

Comment:
Please delete this. Was this a typo

Reply:
ok, think so because i didn't edit that file